database/migrations/2019_08_26_144014_create_bulk_actions_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateBulkActionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('bulk_actions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('perex')->nullable();
+            $table->date('published_at')->nullable();
+            $table->boolean('enabled')->default(false);
```

Comment:
Why we have published_at and enabled?

Reply:
It's taken from example migration https://docs.getcraftable.com/v3/#/admin-generator?id=admin-generator